deploy/src/main/resources/production-kubernetes-prerequisites/README.md

```diff
@@ -0,0 +1,32 @@
+# Production Kubernetes Prerequisites
+
+The Production Kubernetes Prerequisites deployment supports the deployment of the Job Service Database on Kubernetes. This folder contains the kubernetes environment and template files that are required to deploy the Job Service Database and RabbitMQ.
+
+## Service Configuration
+
+### Kubernetes Template
+The `jobservice-prerequisites-deployment.yaml` template file describes the marathon deployment information required for starting the Job Service Database and RabbitMQ. The template file uses property substitution to get values for configurable properties **required** for service deployment. These properties are configured in the kubernetes environment file `kubernetes.env`.
+
+The `jobservice-prerequisites-persistentvolumeclaim.yaml` template file describes the Kubernetes persistent volume  information required by the Job Service database and RabbitMQ.
+
+### Kubernetes Environment
+The `kubernetes.env` file supports configurable property settings necessary for service deployment. These include:
```

Comment:
This is not all that is in that file.

Reply:
Added rabbit variables that were missing.